cypress/utils/utils.ts

```diff
@@ -115,6 +115,7 @@ import {
     getSpecialMigrationWavesTableSelector,
 } from "../e2e/views/migration-wave.view";
 import { manageCredentials, mavenCredential, sourceCredential } from "../e2e/views/analysis.view";
+import { string } from "@oozcitak/infra";
```

Comment:
Why is this required ?

Reply:
Didn't do that on purpose, will remove it

Reply:
Done